insights/parsers/nmcli.py

```diff
@@ -178,7 +178,19 @@ class NmcliConnShow(CommandParser):
 
     """
     def parse_content(self, content):
-        self.data = parse_delimited_table(content)
+        table_columns = set(["NAME", "UUID", "TYPE", "DEVICE"])
```

Comment:
The following logic can be simplified by utilizing the parameters of the `parse_delimited_table`
See:
https://github.com/RedHatInsights/insights-core/blob/8715825b19890cea227c19f6efbb162b5fc80de8/insights/parsers/__init__.py#L396

Reply:
Done